torch/storage.py

```diff
@@ -237,6 +237,17 @@ def _new_shared(cls, size, *, device='cpu'):
     def untyped(self):
         return self
 
+    def byteswap(self, dtype):
```

Comment:
The mypy error should go away if you move this function def under `UntypedStorage` instead of `_StorageBase`

Reply:
Wait actually, there's a better way to fix the error. Let me find it